ui/imports/shared/views/HistoryView.qml

```diff
@@ -359,24 +358,27 @@ ColumnLayout {
                 let asset = root.walletRootStore.getAssetForSendTx(tx)
 
                 const req = Helpers.lookupAddressesForSendModal(root.walletRootStore.accounts,
-                                                              root.walletRootStore.savedAddresses,
-                                                              tx.sender,
-                                                              tx.recipient,
-                                                              asset,
-                                                              tx.isNFT,
-                                                              tx.amount,
-                                                              tx.chainId)
-
-                root.sendModal.preSelectedAccountAddress = req.preSelectedAccount.address
-                root.sendModal.preSelectedRecipient = req.preSelectedRecipient
-                root.sendModal.preSelectedRecipientType = req.preSelectedRecipientType
-                root.sendModal.preSelectedHoldingID = req.preSelectedHoldingID
-                root.sendModal.preSelectedHoldingType = req.preSelectedHoldingType
-                root.sendModal.preSelectedSendType = req.preSelectedSendType
-                root.sendModal.preDefinedAmountToSend = req.preDefinedAmountToSend
-                root.sendModal.preSelectedChainId = req.preSelectedChainId
-                root.sendModal.onlyAssets = false
-                root.sendModal.open()
+                                                                root.walletRootStore.savedAddresses,
+                                                                tx.sender,
+                                                                tx.recipient,
+                                                                asset,
+                                                                tx.isNFT,
+                                                                tx.amount,
+                                                                tx.chainId)
+                Global.openSendModal(req.preSelectedSendType, //sendType
+                                     req.preSelectedAccount.address, //senderAddress
+                                     req.preSelectedHoldingID, //tokenId
+                                     req.preSelectedHoldingType, //tokenType
+                                     req.preDefinedAmountToSend, //tokenAmount
+                                     req.preSelectedChainId, //chainId
+                                     req.preSelectedRecipient, //recipientAddress
+                                     req.preSelectedRecipientType, //recipientType
+                                     false, //onlyAssets
+                                     true, //interactive
+                                     "", //publicKey
```

Comment:
... and finally you could omit all the empty/default/uninteresting parameters here

Reply:
Very good point! Adding it now..

Reply:
Done @caybro